### PR TITLE
[#174378066] Dialog page stylize

### PR DESCRIPTION
**Pull request by IlgizGilmanov**
Task: [Stylize dialog page](https://www.pivotaltracker.com/story/show/174378066)
Design: https://projects.invisionapp.com/d/#/console/20307299/427358480/preview

**Resolution**
Снимок экрана 2020-09-03 в 20 40 15
Снимок экрана 2020-09-03 в 20 40 34

**Dialog**
Снимок экрана 2020-09-03 в 20 47 28
Снимок экрана 2020-09-03 в 20 47 49
Снимок экрана 2020-09-03 в 20 48 09

### CODE REVIEW

File: assets/javascripts/discourse/templates/list/topic-list-item.hbr [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    {{~/if}}
+  </span>
+  <div class="link-bottom-line">
+    {{#unless hideCategory}}
+      {{#unless topic.isPinnedUncategorized}}
+        {{category-link topic.category}}
+      {{/unless}}
+    {{/unless}}
+    {{!-- custom code --}}
+    {{#if true}}
```
Comment (ShaidullinArtur) [inline assets/javascripts/discourse/templates/list/topic-list-item.hbr]: strange condition

**Review by ShaidullinArtur — APPROVED** [review]
locally spotted a few bugs, but I think that we can fix them in a separate PR

File: assets/javascripts/discourse/templates/list/topic-list-item.hbr (hunk 1 above)
Comment (IlgizGilmanov) [reply assets/javascripts/discourse/templates/list/topic-list-item.hbr]: PR is not ready yet, I am waiting for the merge of the #32 to continue, that's why such conditions :)

**Review by ShaidullinArtur — APPROVED** [review]

File: assets/javascripts/discourse/components/suggested-dialog-list.js.es6 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+	classNames: ["suggested-dialog-list"]
```
Comment (DmitryBarskov) [inline assets/javascripts/discourse/components/suggested-dialog-list.js.es6]:
```suggestion
  classNames: ["suggested-dialog-list"]
```

**Review by DmitryBarskov — APPROVED** [review]

**Review by Vadim-Kurnatovskiy — APPROVED** [review]

**Pull request merged**